db/seeds.rb

```diff
@@ -70,6 +72,36 @@ def create_paperworks
   puts "Created #{NUM_PAPERWORKS} Paperworks"
 end
 
+def create_template_action_items
+  1.upto(NUM_TEMPLATE_ACTION_ITEMS) do |i|
```

Comment:
is there a reason for using a loop with the "i" variable? we can do something like 

`10.times { puts "hello" }` or in this case `NUM_TEMPLATE_ACTION_ITEMS.times { puts "hello" }`